vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_wNa16.py

```diff
@@ -63,11 +61,12 @@ def create_weights(self, layer: torch.nn.Module, input_size: int,
                        input_size_per_partition: int,
                        params_dtype: torch.dtype, weight_loader: Callable,
                        **kwargs):
+
         output_size_per_partition = sum(output_partition_sizes)
 
         # If group_size is -1, we are in channelwise case.
-        channelwise = (self.group_size == -1)
-        group_size = input_size if channelwise else self.group_size
+        channelwise = self.group_size == -1
+        group_size = self.group_size if self.group_size != -1 else input_size
```

Comment:
why change this?

Reply:
the second condition is just clearer as what the group_size is and why